plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -50,6 +51,9 @@
     private int fetchRetryCount = 2;
     private int nonAggregateLimitForBrokerQueries = 25_000;
     private int maxRowsPerSplitForSegmentQueries = 50_000;
+    private int maxRowsForBrokerQueries = 50_000;
+    private boolean aggregationPushdownEnabled = true;
+    private boolean distinctCountPushdownEnabled = true;
```

Comment:
nit: `distinctCountPushdownEnabled` -> `countDistinctPushdownEnabled` (since that's what the optimizer calls it)